product_docs/docs/pgd/5/deploy-config/deploy-tpa/deploying/01-configuring.mdx

```diff
@@ -104,7 +104,7 @@ Optionally, use `--edb-repositories repository …` to specify EDB repositories
 
 
 ### Software versions
-By default, TPA uses the latest major version of Postgres. Specify `--postgres-version` to install an earlier supported major version, or specify both version and distribution using one of the flags described under [Configure](#configure).
+By default, TPA uses the latest major version of Postgres. Specify `--postgres-version` to install an earlier supported major version, or specify both version and distribution using one of the flags described under [Configure](#).
```

Comment:
This one doesn't work per se, I need to add in a new heading or anchor higher up...